.circleci/config.yml

```diff
@@ -7,12 +7,21 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - yarn-packages-v5-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - yarn-packages-v6-{{ .Branch }}-{{ checksum "yarn.lock" }}
       - run:
           name: yarn-install
           command: yarn install --frozen-lockfile
+      - run:
+          name: yarn-build
+          command: yarn workspaces run build
+      - run:
+          name: refresh-node-modules
+          command: rm -rf "node_modules"
+      - run:
+          name: yarn-install-with-dist-folders
+          command: yarn install --frozen-lockfile
```

Comment:
Ok so... it looks stupid but!

`yarn install` with `workspaces` will copy the content of `packages` into `node_modules/@fwl/`, that's how it does the link between the different requirement of packages.

It's supposed to be a symlink (which would mean that `./node_modules/@fwl/tracing` points to `./packages/tracing` but it's not, it's a copy (and that's a choice I **really** don't understand).

The problem is that when running tests on database (for instance), it tries to import something from `./node_modules/@fwl/tracing` BUT if we don't build tracing before that, jest try to work with the typescript file, which has illegal syntax (ESM and all that 😩 ). 

Soo... since I can't find any other way right now to combate that, I can install packages, build packages, remove node_modules and reinstall. This will copy the `dist` folder into the `node_modules/@fwl` packages.

Reply:
> but it's not, it's a copy (and that's a choice I really don't understand)

Maybe to avoid some problems when working with some docker images (like using an old windows OS in it?) or if something changes with node and its file URLs? Or to avoid breaking with some configs? 🤔 

> Soo... since I can't find any other way right now to combate that, I can install packages, build packages, remove node_modules and reinstall. This will copy the dist folder into the node_modules/@fwl packages.

That's a strange solution, but OK 😄

Reply:
> Maybe to avoid some problems when working with some docker images (like using an old windows OS in it?) or if something changes with node and how it parses files? Or to avoid breaking with some configs?

Honestly no idea